addon/components/polaris-popover.js

```diff
@@ -117,46 +116,41 @@ export default Component.extend({
    * @default noop
    * @public
    */
-  onClose() {},
+  onClose() {}
 
   /**
    * @private
    */
-  verticalPosition: computed('preferredPosition', {
-    // If `preferredPosition` is set to `mostSpace`, the value
-    // will be calculated and set when the user opens the popover.
-    // The only allowed values are 'above' and 'below', so we
-    // return null for anything other than those values and let
-    // ember-basic-dropdown use its default value.
```

Comment:
Curious if there was a reason for dropping this comment

Reply:
I didn't realize it got dropped 🤔I want to blame a codemod but that doesn't sound right.. I'll add it back.